cloud/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.9-slim AS base
```

Comment:
Do we need python?

here is pulumi's base dockerfile sans SDKs:
https://github.com/pulumi/pulumi-docker-containers/blob/main/docker/base/Dockerfile

I don't think we need the SDKs when we're using the automation API. Excluding unnecessary runtimes should save some space